server.js

```diff
@@ -1,5 +1,96 @@
-// import your node modules
+const express = require('express')
```

Comment:
Try to be consistent with semi-colons. Either omit or keep them, stick with one. :)